lib/galaxy/datatypes/converters/fastagz_to_fasta.xml

```diff
@@ -0,0 +1,11 @@
+<tool id="CONVERTER_fastqgz_to_fastq" name="Convert fastq.gz files to fastq" version="1.0.0" hidden="true">
```

Comment:
The id and the name needs to be adopted to FASTA

Reply:
done